lib/board.rb

```diff
@@ -0,0 +1,49 @@
+class Board
+attr_accessor :cells
+
+  def initialize
+    reset!
+  end
+
+  def reset!
+    @cells = Array.new(9, " ")                                                                                            
+  end
+
+  def cells
+    @cells
```

Comment:
With `attr_accessor :cells`, this method is not necessary. The `attr_accessor` adds two (hidden) methods:

```
def cells
  @cells
end
```
and
```
def cells=(cells)
  @cells = cells
end
```